Libraries/LibCrypto/ASN1/DER.cpp

```diff
@@ -265,7 +265,7 @@ ErrorOr<void> Encoder::write_length(size_t value)
 
     double minimum_bits = AK::log2(value);
     size_t size_in_bits = AK::floor(minimum_bits) + 1;
-    size_t size = ceil_div(size_in_bits, 8ul);
+    size_t size = ceil_div(size_in_bits, 8z);
```

Comment:
I'm just throwing the idea here, you probably want to wait for maintainer input on that.
I think that we usually prefer types with an explicit size rather than `unsigned long` or `size_t`. Shouldn't we do the same thing for int litterals and introduce something like `_u64` for this situation?

Something like:
```
u64 operator""_u64(unsigned long long int u){
    return u;
}
```

Reply:
I think in this case `uz` is the better option, as the other argument of `ceil_div` is a `size_t`.